backend/handlers/report.js

```diff
@@ -0,0 +1,161 @@
+        if (!exists)
+          return responseUtil.Build(500, { Message: "Invalid ID for the user making the report!" });
+
+        let exists2 = await AccountAPI.Get(request.AccuserID);
+        if(!exists2)
+          return responseUtil.Build(500, { Message: "Invalid ID for the reported user!" });
+
+        let report = await ReportAPI.Create(request);
+
+        if (report) {
+          let result = {
+              Message: "Report created!",
+              Game: request
+          };
+          return responseUtil.Build(200, result);
+        } else return responseUtil.Build(500, { Message: "Report Save Failed" });
+
+    } catch (err) {
+        console.log(err);
+        return responseUtil.Build(500, { Message: err.message });
+    }
+},
+
+// // REPORT A NEW ACCOUNT //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//   NewReport: async function (event) {
+//     try {
+//       // if nothing was provided in the request, return a 204 HTTPS code (No content)
+//       if (!event)
+//         return responseUtil.Build(204, { Message: "No information was provided with the request!" });
+
+//       // next parse the request
+//       let request = JSON.parse(event.body);
+
+//       // they need to have entered an email, username, and a password
+//       if (!request.ReportedAccountID)
+//         throw new Error("Must include the ID of the account to be reported!");
+
+//       if (!request.AccuserID)
+//         throw new Error("Must include the ID of the account who is making the report!");
+
+//       // let's have a create date for our new report as well as an ID for the report
+//       request.CreateDate = moment().toISOString();
+//       request.ID = shortid.generate();
+
+//       // check to make sure that this ID is not already used
+//       while(ReportAPI.Get(request.ID))
+//         request.ID = shortid.generate();
+
+//       // validate that both accounts exist
+//       let exists = await AccountAPI.Get(request.ReportedAccountID);
+//       if (!exists)
+//         throw new Error("Invalid ID for the user making the report!");
+
+//       let exists2 = await AccountAPI.Get(request.AccuserID);
+//       if(!exists2)
+//         throw new Error("Invalid ID for the reported user!");
+
+//       // save to the database
+//       let dynamoDB = new AWS.DynamoDB.DocumentClient({ apiVersion: "2012-08-10" }); // connect to the database
+
+//       const parameters = {
+//         TableName: tableName,
+//         Item: request,
+//       };
+
+//       let NewReport = await dynamoDB.put(parameters).promise();
+
+//       let result = {
+//         Message: "Report created!",
+//         Report: NewReport
+//       };
+
+//       return responseUtil.Build(200, result);
+//     } catch (err) {
+//       console.log(err);
+//       return responseUtil.Build(500, { Message: err.message });
+//     }
+//   },
```

Comment:
Looks like an unnecessary commented version of the previous function, just without the call to the API. Probably fine to remove.